python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -440,8 +459,8 @@ class RandomForestRegressor(Base):
                  'accuracy_metric']
 
     def __init__(self, n_estimators=10, max_depth=-1, handle=None,
-                 max_features=1.0, n_bins=8,
-                 split_algo=0, split_criterion=2,
+                 max_features=None, n_bins=8,
```

Comment:
Similar update to `max_features` default value as above.

Also, `split_algo` should be 1 (`GLOBAL_QUANTILE`) not 2. Maybe also update it in the `__cinit__`  above for consistency?